src/components/atoms/Button/index.jsx

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line no-restricted-exports
```

Comment:
You can add this rule to `.eslintrc.json` to avoid eslint error here
```
"no-restricted-exports": [1, {
      "restrictedNamedExports": [
        "then"
      ]
    }]
```